test/test_autograd.py

```diff
@@ -8838,7 +8838,7 @@ def chain_with_only_parent_view_func(x):
                     x = x.split_with_sizes([1, 3], -1)[0]
 
                 with torch.autograd._force_original_view_tracking(False):
-                    x = x.chunk(2, -1)
+                    x = x.chunk(2, 0)
```

Comment:
ah yeah, I realized that the original test didn't make a lot of sense since tries to chunk a size 1 dim into 2 parts. This change has it operate on a non-1-size dim

Reply:
But why change this one but not the ones above? :D

Reply:
good point, changing the one above too